### PR TITLE
cypress: improved stability

### PR DESCRIPTION
**Pull request by dctalbot**
- [x] Identified the issue which this PR solves.
- [x] Read the [**CONTRIBUTING**](https://github.com/target/goalert/blob/master/CONTRIBUTING.md) document.
- [x] Code builds clean without any errors or warnings.
- [x] Added appropriate tests for any new functionality.
- [x] All new and existing tests passed.
- [x] Added comments in the code, where necessary.
- [x] Ran `make check` to catch common errors. Fixed any that came up.

**Description:**
This PR refactors the cypress tests for stability by:
1 - scrolling the no-coverage checkbox into view before confirming visibility
2- accounts for weekend boundary cases that result in multiple calendar events (wrap saturday/sunday)

### CODE REVIEW

File: web/src/cypress/integration/temporarySchedule.ts [hunk 1]
```diff
@@ -137,6 +138,7 @@ function testTemporarySchedule(screen: string): void {
       cy.get('[data-cy="shifts-list"]').should('contain', manualAddUser.name)
       cy.dialogClick('Submit')
       cy.get('[data-cy="no-coverage-checkbox"]')
+        .scrollIntoView()
         .should('be.visible')
```
Comment (mastercactapus) [inline web/src/cypress/integration/temporarySchedule.ts:142]: Can we get rid of both of these?

`scrollIntoView` should happen automatically on actions click check/click as well as asserting/waiting for visibility of the input field.

Unless I'm mistaken `cy.get("input[name=allowCovoerageGaps]").check()` should handle retry, waiting, scrolling, and ensuring it's visible.

Comment (dctalbot) [reply web/src/cypress/integration/temporarySchedule.ts:142]: Oh yes, I think you're right. I will test...

**Review by mastercactapus — APPROVED** [review]

**Review by KatieMSB — APPROVED** [review]

**Review by mastercactapus — APPROVED** [review]
